packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -1897,7 +1897,7 @@ You can enroll, activate, manage, and verify factors inside the authentication c
 ### Enroll Factor
 
 
-<ApiOperation method="post" url="/api/v1/authn/factors" />
+<ApiOperation method="post" url="/api/v1/authn/factors" /> <SupportsCors />
```

Comment:
Is there an example of this label anywhere in the doc. or is this the first candidate.

Reply:
https://developer.okta.com/docs/reference/api/users/#get-user